lib/arel/visitors/to_sql.rb

```diff
@@ -131,6 +131,7 @@ def visit_Arel_Nodes_InsertStatement o, collector
         end
 
         if o.values
+          collector << " VALUES"
```

Comment:
This conflicts with `empty_insert_statement_value` in Active Record.

https://travis-ci.org/rails/rails/jobs/228871658#L407
https://github.com/rails/rails/blob/v5.1.0/activerecord/lib/active_record/relation.rb#L56
https://github.com/rails/rails/blob/v5.1.0/activerecord/lib/active_record/connection_adapters/abstract/database_statements.rb#L322-L324